dfpl/options.py

```diff
@@ -1034,141 +1226,151 @@ def parseTrainGnn(parser: argparse.ArgumentParser) -> None:
-        "numbered in the order of their appearance in the input file."
-        "A header is expected and respective column names are used.",
-        default=argparse.SUPPRESS,
+        help="Path to a .json file containing arguments. Any arguments present in the config"
+        "file will override arguments specified via the command line or by the defaults.",
     )
     files_args.add_argument(
-        "-o",
-        "--outputDir",
-        metavar="DIR",
+        "--test_path",
         type=str,
-        help="Prefix of output directory. It will contain a log file and the file specified"
-        "with --outputFile.",
-        default=argparse.SUPPRESS,
+        help="Path to CSV file containing testing data for which predictions will be made.",
     )
     files_args.add_argument(
-        "--outputFile",
-        metavar="FILE",
+        "--preds_path",
         type=str,
-        help="Output .CSV file name which will contain one prediction per input line. "
-        "Default: prefix of input file name.",
-        default=argparse.SUPPRESS,
+        help="Path to CSV or PICKLE file where predictions will be saved.",
     )
-    general_args.add_argument(
-        "-t",
-        "--type",
-        metavar="STR",
+    files_args.add_argument(
+        "--calibration_path",
         type=str,
-        choices=["fp", "smiles"],
-        help="Type of the chemical representation. Choices: 'fp', 'smiles'.",
-        default=argparse.SUPPRESS,
+        help="Path to data file to be used for uncertainty calibration.",
     )
-    general_args.add_argument(
-        "-k",
-        "--fpType",
-        metavar="STR",
+    files_args.add_argument(
+        "--calibration_features_path",
         type=str,
-        choices=["topological", "MACCS"],  # , 'atompairs', 'torsions'],
-        help="The type of fingerprint to be generated/used in input file.",
-        default=argparse.SUPPRESS,
+        nargs="+",
+        help="Path to features data to be used with the uncertainty calibration dataset.",
     )
+    files_args.add_argument("--calibration_phase_features_path", type=str, help="")
     files_args.add_argument(
-        "--ecModelDir",
+        "--calibration_atom_descriptors_path",
         type=str,
-        metavar="DIR",
-        help="The directory where the full model of the encoder will be saved (if trainAE=True) or "
-        "loaded from (if trainAE=False). Provide a full path here.",
-        default=argparse.SUPPRESS,
+        help="Path to the extra atom descriptors.",
     )
     files_args.add_argument(
-        "--fnnModelDir",
+        "--calibration_bond_descriptors_path",
         type=str,
-        metavar="DIR",
-        help="The directory where the full model of the fnn is loaded from. "
-        "Provide a full path here.",
-        default=argparse.SUPPRESS,
+        help="Path to the extra bond descriptors that will be used as bond features to featurize a given molecule.",
```

Comment:
For paths it would be good if the help would mention a format.

```suggestion
        help="Extra bond descriptors file. Path to the extra bond descriptors that will be used as bond features to featurize a given molecule.",
```